docs/visualization/plotting-overview.ipynb

```diff
@@ -308,6 +308,58 @@
    "source": [
     "(100 * (da1 - da1.min())).plot(norm='log')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Migrating to the New Backend\n",
+    "\n",
+    "Since version 0.17, Scipp can use either the old builtin plotting backend or [Plopp](https://scipp.github.io/plopp/).\n",
+    "The old backend is deprecated and will be removed in the first release in or after August 2023.\n",
+    "It is recommended to switch to the new backend already now to avoid breaking your code through an update later.\n",
+    "There are 3 ways of doing so:\n",
```

Comment:
I think I would revert the order of ways you list below?

Reply:
I'm not so sure. The config setting is causing me trouble because it means that there always is some code that doesn't work until every last piece is migrated. And it messes with things like automated tests and docs builds.

Reply:
What do settings the user does have to do with automated tests and docs builds?

Reply:
I thought it is clear that plopp will be the default in the future?

> What do settings the user does have to do with automated tests and docs builds?

KI meant 'user' as in developers of downstream packages. Their dev setup would depend on the concrete dev's environment.

Reply:
I do not think Scipp's documentation should be geared primarily towards downstream developers.

Reply:
My idea was that `plopp` is default, for sure.